ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -1297,3 +1298,83 @@ def test_advanced_funnel_multiple_exclusions_between_steps(self):
         self.assertCountEqual(
             self._get_people_at_step(filter, 1), [person4.uuid],
         )
+
+    def test_funnel_with_elements_chain(self):
+        person1 = _create_person(distinct_ids=["test"], team_id=self.team.pk)
+        _create_event(team=self.team, event="user signed up", distinct_id="test")
+        _create_event(
+            team=self.team,
+            event="$autocapture",
+            distinct_id="test",
+            properties={"$current_url": "http://example.com/something_else"},
+            elements=[Element(tag_name="img"), Element(tag_name="svg")],
+        )
+
+        person2 = _create_person(distinct_ids=["test2"], team_id=self.team.pk)
+        _create_event(team=self.team, event="user signed up", distinct_id="test2")
+
+        filters = {
+            "events": [
+                {"id": "user signed up", "type": "events", "order": 0,},
+                {
+                    "id": "$autocapture",
+                    "name": "$autocapture",
+                    "order": 1,
+                    "properties": [{"key": "tag_name", "value": ["img"], "operator": "exact", "type": "element"}],
+                    "type": "events",
+                },
+            ],
+            "insight": INSIGHT_FUNNELS,
+        }
+
+        filter = Filter(data=filters)
+        funnel = ClickhouseFunnel(filter, self.team)
+
+        result = funnel.run()
```

Comment:
There's a whole lot of intermediary variables here that are not used.

You could get rid of `filters`, `filter` and `funnel` assignments and do it all on one line here. Given multiple tests exist here this would be more obvious even

Reply:
Nitpick but still valid, doesn't block merge though :)

Reply:
filter is used in _get_people_for_step, and I got rid of `funnel`